src/main/java/edu/kis/powp/jobs2d/drivers/SelectDriverMenuOptionListener.java

```diff
@@ -1,23 +1,46 @@
 package edu.kis.powp.jobs2d.drivers;
 
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.appbase.Application;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.LineDrawerAdapter;
 import edu.kis.powp.jobs2d.features.DriverFeature;
 
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
 public class SelectDriverMenuOptionListener implements ActionListener {
-	private DriverManager driverManager;
-	private Job2dDriver driver = null;
-
-	public SelectDriverMenuOptionListener(Job2dDriver driver, DriverManager driverManager) {
-		this.driverManager = driverManager;
-		this.driver = driver;
-	}
-
-	@Override
-	public void actionPerformed(ActionEvent e) {
-		driverManager.setCurrentDriver(driver);
-		DriverFeature.updateDriverInfo();
-	}
+    private DriverManager driverManager;
+    private Job2dDriver driver = null;
+    private Application application;
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.